iroh/src/protocol.rs

```diff
@@ -300,12 +302,28 @@ impl RouterBuilder {
                                     break;
                                 }
                             }
-                            Some(Ok(Err(inner))) => {
-                                debug!("Task errored: {inner:?}");
+                            Some(Ok(Some(()))) => {
+                                debug!("Task finished");
```

Comment:
What is the granularity of tasks here?  Basically I'm wondering whether this is too nosy to be on debug level and should rather be on trace level.

Reply:
One task per incoming connection